lib/node_modules/@stdlib/repl/README.md

```diff
@@ -71,6 +71,7 @@ The function accepts the following `options`:
 -   **outputPrompt**: output prompt. If the output prompt includes the character sequence `%d`, the output prompt includes line numbers. Default: `'Out[%d]: '`.
 -   **welcome**: welcome message.
 -   **padding**: number of empty lines between consecutive commands. Default: `1`.
+-   **themes**: table mapping of color themes to load for syntax highlighting.
```

Comment:
```suggestion
-   **themes**: table containing color themes for syntax highlighting.
```

Reply:
"load" is an overloaded term and often used to mean loading from disk. Would be good to avoid this ambiguity.